pages/index/monitoring.vue

```diff
@@ -34,6 +34,28 @@
             <MonitoringFilter @onSearch="search()" />
         </div>
         <v-divider />
+        <div v-if="showFeatures" class="px-4">
+            <p class="font-weight-regular pt-2">Legenda:</p>
+            <v-col>
+                <v-row class="mb-2">
+                    <v-icon class="mr-2" color="#990099">mdi-square</v-icon>
+                    Desmatamento em Regeneração
+                </v-row>
+                <v-row class="mb-2">
+                    <v-icon class="mr-2" color="#b35900">mdi-square</v-icon>
+                    Fogo em FLoresta
```

Comment:
Why dont you call it from a `data object` on vue?

Reply:
Take a look, it is with `FLoresta`

Reply:
I'm following de legend pattern of the other components.